docs/controls/Line.md

```diff
@@ -183,6 +183,7 @@ Refer [Graph](../core/Graph.md) `Root` for more details.
 | onClick       | Function | undefined          | Any action that can be performed when clicking on the data point             |
 | style         | object   | {}                 | Any style that can be applied. Refer [Styles](#style)                        |
 | legendOptions | object   | undefined          | Option to show/hide shape and/or line. Refer [LegendOptions](#legendOptions) |
+| showShapes    | boolean  | undefined          | Option to show/hide shapes per data set                                      |
```

Comment:
Should we make it clear in the documentation whether this overrides the graph option?

Reply:
Updated the documentation [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/cd940e1ad08350d7f59d325884f691571345033a).